recipe/bld.bat

```diff
@@ -1,20 +1,35 @@
-COPY conda_src\conda\core\path_actions.py "%SP_DIR%\conda\core\path_actions.py"
-COPY conda_src\conda\utils.py "%SP_DIR%\conda\utils.py"
+@ECHO on
 
-:: we need these for noarch packages with entry points to work on windows
-COPY conda_src\conda\shell\cli-%ARCH%.exe entry_point_base.exe
+RENAME "%SP_DIR%\conda\core\path_actions.py" path_actions.py.bak || goto :error
+COPY conda_src\conda\core\path_actions.py "%SP_DIR%\conda\core\path_actions.py" || goto :error
+RENAME "%SP_DIR%\conda\utils.py" utils.py.bak || goto :error
+COPY conda_src\conda\utils.py "%SP_DIR%\conda\utils.py" || goto :error
+RENAME "%SP_DIR%\conda\deprecations.py" deprecations.py.bak || goto :error
+COPY conda_src\conda\deprecations.py "%SP_DIR%\conda\deprecations.py" || goto :error
 
-COPY menuinst_src\menuinst\__init__.py "%SP_DIR%\menuinst\__init__.py"
-COPY menuinst_src\menuinst\win32.py "%SP_DIR%\menuinst\win32.py"
+:: we need these for noarch packages with entry points to work on windows
+COPY "conda_src\conda\shell\cli-%ARCH%.exe" entry_point_base.exe || goto :error
 
 :: This is ordinarily installed by the installer itself, but since we are building for a
 :: standalone and have only an env, not an installation, include it here.
-COPY constructor\constructor\nsis\_nsis.py "%PREFIX%\Lib\_nsis.py"
+COPY constructor_src\constructor\nsis\_nsis.py "%PREFIX%\Lib\_nsis.py" || goto :error
+
+pyinstaller --clean --log-level=DEBUG src\conda.exe.spec || goto :error
+MKDIR "%PREFIX%\standalone_conda" || goto :error
+MOVE dist\conda.exe "%PREFIX%\standalone_conda\conda.exe" || goto :error
+
+:: Collect licenses
+%PYTHON% src\licenses.py ^
+  --prefix "%BUILD_PREFIX%" ^
+  --include-text ^
+  --text-errors replace ^
+  --output "%SRC_DIR%\3rd-party-licenses.json" || goto :error
 
-pyinstaller conda.exe.spec --log-level DEBUG
-if %ErrorLevel% neq 0 exit \b 1
+RD /s /q "%PREFIX%\lib" || goto :error
 
-MKDIR "%PREFIX%\standalone_conda"
-MOVE dist\conda.exe "%PREFIX%\standalone_conda\conda.exe"
+goto :EOF
 
-RD /s /q "%PREFIX%\lib"
+:error
+set "exitcode=%errorlevel%"
+echo Failed with error #%exitcode%.
+exit /b %exitcode%
```

Comment:
Please add line breaks at the end of files. This is just causing unnecessary diffs compared to conda-forge